exchange/auction.go

```diff
@@ -87,18 +122,30 @@ func (a *auction) doCache(ctx context.Context, cache prebid_cache_client.Client,
 			if vast && topBidPerBidder.bidType == openrtb_ext.BidTypeVideo {
 				vast := makeVAST(topBidPerBidder.bid)
 				if jsonBytes, err := json.Marshal(vast); err == nil {
```

Comment:
I think we should also append `err` from `json.Marshal` to `errs`, no? For both, this and the one up on line 109.